src/nvme/ioctl.c

```diff
@@ -2034,13 +2036,14 @@ int nvme_resv_report(int fd, __u32 nsid, bool eds, __u32 len,
 }
 
 int nvme_zns_mgmt_send(int fd, __u32 nsid, __u64 slba,
-		       enum nvme_zns_send_action zsa,
-		       bool select_all, __u32 data_len,
+		       enum nvme_zns_send_action zsa, bool select_all,
+			   __u8 zsaso, __u32 data_len,
```

Comment:
whitepace damage in front of zsaso

Reply:
Thanks for the details, I didnt realize that above line using both(space & tab).
fixed with spaces